packages/gatsby-theme-docs/src/components/release-note-heading.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { designSystem } from '@commercetools-docs/ui-kit';
+
+const Container = styled.div`
+  font-weight: ${designSystem.typography.fontWeights.bold};
+  line-height: ${designSystem.typography.lineHeights.releaseNoteHeader};
+  font-size: ${designSystem.typography.fontSizes.h3};
+  color: unset;
+  margin: ${(props) =>
+      props.as === 'h1'
+        ? designSystem.dimensions.spacings.l
+        : designSystem.dimensions.spacings.huge}
+    0 0 !important;
```

Comment:
Shouldn't it be the other way around? Also, `important` should not be needed.

```suggestion
      props.as === 'h1'
        ? designSystem.dimensions.spacings.huge
        : designSystem.dimensions.spacings.l}
    0 0;
```

Reply:
> Shouldn't it be the other way around? Also, `important` should not be needed.

No it's not the other way around, see the designs:

- https://commercetools.invisionapp.com/d/main?origin=v7#/console/19438556/408192223/inspect?scrollOffset=215
- https://commercetools.invisionapp.com/d/main?origin=v7#/console/19438556/412794930/inspect?scrollOffset=187

`important` is needed because we have `important` on margin styling of the parent component.

Reply:
> No, it's not the other way around, see the designs:

My bad, I confused that with the font size. All good.

> important is needed because we have important on margin styling of the parent component for the page title

Got it, thanks.